go-controller/pkg/metrics/metrics.go

```diff
@@ -70,6 +70,14 @@ type stopwatchStatistics struct {
 	longTermAvg    string
 }
 
+// MetricResourceMaxRetryFailed is the number event handling retries resources reached its maximum limit.
+var MetricResourceMaxRetryFailed = prometheus.NewCounter(prometheus.CounterOpts{
+	Namespace: MetricOvnkubeNamespace,
+	Subsystem: MetricOvnkubeSubsystemMaster,
+	Name:      "resource_max_retry_failed",
```

Comment:
You must follow prometheus naming standards: https://prometheus.io/docs/practices/naming/

How about `resource_retry_failures_total` ?
Naming is hard :) Happy to hear something better..